dataflow/flex-templates/streaming_beam/README.md

```diff
@@ -0,0 +1,282 @@
+# Dataflow flex templates - Streaming Beam
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor)
+
+📝 Docs: [Using Flex Templates](https://cloud.google.com/dataflow/docs/guides/templates/using-flex-templates)
+
+Samples showing how to create and run an
+[Apache Beam](https://beam.apache.org/) template with a custom Docker image on
+[Google Cloud Dataflow](https://cloud.google.com/dataflow/docs/).
+
+## Before you begin
+
+Follow the
+[Getting started with Google Cloud Dataflow](../README.md)
+page, and make sure you have a Google Cloud project with billing enabled
+and a *service account JSON key* set up in your `GOOGLE_APPLICATION_CREDENTIALS`
+environment variable.
+Additionally, for this sample you need the following:
+
+1. Create a
+    [Cloud Storage bucket](https://cloud.google.com/storage/docs/creating-buckets).
+
+    ```sh
+    export BUCKET="your-gcs-bucket"
+    gsutil mb gs://$BUCKET
+    ```
+
+1. Create a
+    [Pub/Sub topic](https://cloud.google.com/pubsub/docs/admin#creating_a_topic)
+    and a
+    [subscription](https://cloud.google.com/pubsub/docs/admin#creating_subscriptions)
+    to that topic.
+    This is a streaming source of data for the sample.
+
+    ```sh
+    # For simplicity we use the same topic name as the subscription name.
+    export TOPIC="messages"
+    export SUBSCRIPTION="$TOPIC"
+
+    gcloud pubsub topics create $TOPIC
+    gcloud pubsub subscriptions create --topic $TOPIC $SUBSCRIPTION
+    ```
+
+1. Create a
+    [Cloud Scheduler job](https://cloud.google.com/scheduler/docs/quickstart)
+    to publish "positive" and "negative" ratings every
+    [1 and 2 minutes](https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules).
+    This publishes messages to the Pub/Sub source topic.
+
+    ```sh
+    # Create a publisher for "positive ratings" that publishes 1 message per minute
+    # If an App Engine app does not exist for the project, this step will create one.
+    gcloud scheduler jobs create pubsub thumbs-up-publisher \
```

Comment:
Can you rename `thumbs-up-publisher` to `positive-ratings-publisher` and `thumbs-down-publisher` to `negative-ratings-publisher`?

Make sure to rename in the cleanup section as well. Thanks!

Reply:
Done